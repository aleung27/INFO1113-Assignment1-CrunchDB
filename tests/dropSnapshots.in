SET j 9
SNAPSHOT
SET b 1
SNAPSHOT
DROP 1
LIST SNAPSHOTS
ROLLBACK 2
LIST ENTRIES
LIST SNAPSHOTS
