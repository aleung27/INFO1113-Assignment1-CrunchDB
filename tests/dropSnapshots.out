> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> 2

> ok

> b [1]
j [9]

> 2

